src/connectors/kcrw.js

```diff
@@ -17,14 +16,25 @@ Connector.getTrackInfo = () => {
 			artist: artistText,
 			track: trackText,
 		};
+	}
 
-	} else if (!Util.hasElementClass(metaSelector, 'music') && artistText && showNameText.includes('Today\'s Top Tune')) {
-		return Util.splitArtistTrack(artistText, [': ']);
+	if (!Util.hasElementClass(metaSelector, 'music') && artistText && showNameText === 'Today\'s Top Tune') {
+		const artistTrackSplit = artistText.split(': ');
+		return {
+			artist: artistTrackSplit[0],
+			track: artistTrackSplit[1],
+			currentTime: Util.getSecondsFromSelectors(`${timeSelector} .progress`),
+			duration: Util.getSecondsFromSelectors(`${timeSelector} .total`),
+		};
 	}
 
 	return null;
 };
 
+Connector.isPlaying = () => Util.hasElementClass('button.play', 'active');
+
 Connector.applyFilter(filter);
 
-Connector.isPlaying = () => Util.hasElementClass('button.play', 'active');
+function removeFancyQuotes(text) {
+	return text.replace(/[‘’“”]/g, ''); // filter for Today's Top Tune
```

Comment:
Wonder how useful this could be to moved into https://github.com/web-scrobbler/metadata-filter at somepoint? 

That lib needs some <3

Reply:
Sure, I haven't looked into adding anything to that yet, but if you think it would be useful, I'll take a look.